packages/api/amplify_api/example/integration_test/graphql/iam_test.dart

```diff
@@ -417,7 +442,10 @@ void main({bool useExistingTestUser = false}) {
 
       testWidgets('should fetch model that allows guest access',
           (WidgetTester tester) async {
-        final req = ModelQueries.list<Blog>(Blog.classType);
+        final req = ModelQueries.list<Blog>(
+          Blog.classType,
+          authorizationMode: APIAuthorizationType.iam,
```

Comment:
It's unclear why this is necessary now, but it appears to be more accurate since these models have multiple authentication modes and an explicit mode should have been defined from the start.